website/docs/cloud-docs/users-teams-organizations/single-sign-on/entra-id.mdx

```diff
@@ -64,14 +64,14 @@ For more information on the listed features, visit the [Microsoft Entra ID SAML
 1. In the **Add Assignment** dialog, click the **Assign** button.
 
 ## Configuration (HCP Terraform)
-To edit your Azure SSO configuration settings:
+To edit your Entra SSO configuration settings:
 
    1. Go to **Public Certificate**.
    1. Paste the contents of the SAML Signing Certificate you downloaded from Microsoft Entra ID.
    1. Save Settings.
 1. [Verify](/terraform/cloud-docs/users-teams-organizations/single-sign-on/testing) your settings and click "Enable".
 
-1. Your Azure SSO configuration is complete and ready to [use](/terraform/cloud-docs/users-teams-organizations/single-sign-on#signing-in-with-sso).
+1. Your Entra SSO configuration is complete and ready to [use](/terraform/cloud-docs/users-teams-organizations/single-sign-on#signing-in-with-sso).
 
    ![sso-settings](/img/docs/settings-azure.png)
```

Comment:
this image is also a little out of date...

current doc:
![image](https://github.com/user-attachments/assets/8418579a-9112-457d-a414-3d4d4762a928)


current product: 

![image](https://github.com/user-attachments/assets/3a19d9c0-091d-43bc-9be1-c86bfa937396)